packages/date-picker/src/vaadin-lit-month-calendar.js

```diff
@@ -59,7 +59,9 @@ class MonthCalendar extends MonthCalendarMixin(ThemableMixin(PolylitMixin(LitEle
                   ${this.__getWeekNumber(week)}
                 </td>
                 ${week.map((date) => {
-                  const isFocused = dateEquals(date, this.focusedDate);
+                  const isFocused =
+                    dateEquals(date, this.focusedDate) && (this.__hasFocus || dateEquals(date, this.enteredDate));
```

Comment:
nit: this logic is the same for both the Polymer and Lit versions of this component and it could potentially be moved to the mixin since its complexity is higher than the previous logic.

Reply:
Good point. I also wondered why the part generation wasn't added to the mixin. I'll improve this in a separate PR.